walkeeper/README

```diff
@@ -1,24 +1,64 @@
-# WAL safekeeper
+# WAL service
 
-Also know as the WAL service, WAL keeper or WAL acceptor.
-
-The WAL safekeeper acts as a holding area and redistribution center
-for recently generated WAL. The primary Postgres server streams the
-WAL to the WAL safekeeper, and treats it like a (synchronous)
+The zenith WAL service acts as a holding area and redistribution
+center for recently generated WAL. The primary Postgres server streams
+the WAL to the WAL safekeeper, and treats it like a (synchronous)
 replica. A replication slot is used in the primary to prevent the
-primary from discarding WAL that hasn't been streamed to the
-safekeeper yet.
+primary from discarding WAL that hasn't been streamed to the WAL
+service yet.
+
++--------------+              +------------------+
+|              |     WAL      |                  |
+| Compute node |  ----------> |   WAL Service    |
+|              |              |                  |
++--------------+              +------------------+
+                                     |
+                                     |
+                                     | WAL
+                                     |
+                                     |
+                                     V
+                              +--------------+
+			      |              |
+                              | Pageservers  |
+			      |              |
+			      +--------------+
```

Comment:
I wonder why this looks wobbly on github. Tabs?

Reply:
Yeah. I changed it to all-spaces. Thanks!